tracker/tracker/src/main/app/observer/observer.ts

```diff
@@ -239,6 +240,11 @@ export default abstract class Observer {
       }
       this.app.sanitizer.handleNode(id, parentID, node);
     }
+    if (parentID) {
+      if (this.app.sanitizer.isMaskedContainer(parentID)) {
```

Comment:
Move to the line 241 after `sanitizer.handleNode` so you could remove  unnecessary `parentID`  check